src/app/app.module.ts

```diff
@@ -11,6 +11,7 @@ import { routing } from './app.routing';
 import { OurRiversComponent } from './layouts/our-rivers/our-rivers.component';
 import { GridTableComponent } from './components/grid-table/grid-table.component';
 import { BarChartComponent } from './components/bar-chart/bar-chart.component';
+import {CHART_DIRECTIVES} from 'ng2-charts/ng2-charts';
```

Comment:
space between the variable names and brackets please.

`` { CHART_DIRECTIVES } ``